sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -201,7 +201,8 @@ public HttpPipelineTransport CreateTransport(HttpPipelineTransport currentTransp
             {
                 RecordedTestMode.Live => currentTransport,
                 RecordedTestMode.Record => new RecordTransport(_session, currentTransport, entry => _disableRecording.Value, Random),
-                RecordedTestMode.Playback => new PlaybackTransport(_session, _matcher, _sanitizer, Random),
+                RecordedTestMode.Playback => new PlaybackTransport(_session, _matcher, _sanitizer, Random,
```

Comment:
`EntryRecordModel.DontRecord` in theory should never be set when sending a request in playback mode.

We don't have entry to match it with anyway.

Reply:
Good point.